src/main/kotlin/org/elm/lang/core/lookup/ElmLookup.kt

```diff
@@ -65,9 +66,28 @@ object ElmLookup {
     ): List<T> =
             findByName<T>(name, clientLocation).filter { it.moduleName == module }
 
+    /** Like [findByNameAndModule], but in the case of ambiguity, returns the first match located in the [file] */
+    inline fun <reified T : ElmNamedElement> findFirstByNameAndModule(
+            name: String,
+            module: String,
+            file: ElmFile
+    ): T? {
+        val candidates = findByNameAndModule<T>(name, module, file)
+        return when {
+            candidates.size < 2 -> candidates.firstOrNull()
```

Comment:
I know I wrote this code, and so I have only myself to blame, but the `< 2 ->` part is really gross to read. Would you mind changing this to an `if/else`?